analyzers/src/SonarAnalyzer.CFG/ShimLayer/StatementSyntaxExtensions.cs

```diff
@@ -1,12 +1,12 @@
 ﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
```

Comment:
The code from ShimLayer is imported from Stylecop and we don't want to change it in order to be able to easily update to the latest change. Could you please revert this change?

Probably I forgot to mention this, sorry for that.